apps/frontend/src/components/TimeCardPage/CellTypes/CommentModals/ShowCommentModal.tsx

```diff
@@ -51,10 +53,22 @@ const saveEditedComment = (
   // previous comment edited over so set it to deleted
   prevComment.State = CellStatus.Deleted
   setComments(getAllActiveCommentsOfType(typeOfComment, [...comments, newComment]));
-  // TODO: save to DB
+  
+  var rowToMutate = row.Comment; 
```

Comment:
`rowToMutate` should be renamed here - it's really representing the array of comments that is being edited in the current row, so maybe something like `updatedShiftComments`?

Reply:
3cb60f4e01584f31646f4034bd59fe0868dbc8e7